server/tm_cmds.go

```diff
@@ -133,6 +124,8 @@ against which this app has been compiled.
 	}
 }
 
+// Deprecated: prints the content to the standard output using Legacy Amino
+// TODO: add issue to trace it?
 func printlnJSON(v interface{}) error {
```

Comment:
shell we use protobuf here?

Reply:
Showing the pubkey Any as JSON, as you did in the latest commits, sgtm 👍